Add thank you page (#1837)
EuroPython/website: src/content/pages/thank-you.md

communications, and ensures invoices are submitted:

- Doreen Peace Nangira Wanyama
- Theofanis Petkos
- Raquel Dou
- Aris Nivorlis
- Panagiotis Skias

## Operations ⚙️

This is the largest team, divided into several smaller teams. You might have met
them onsite or online when contacting our helpdesk.

Operations Team:

- Jake Balas
- George Zisopoulos
- Janusz Kamieński
- Liberta Gani
- Mahe Iram Khan
- Moisés Guimarães
- Martin Borus
- Niklas Mertsch
- Panagiotis Kyrillos
- Piotr Gnus
- Raquel Dou
- Çınar Fidanboy
- Aeneas Christodoulou
- Toomas Ormisson
- Szymon Cader
- Jakub Červinka

## Code of Conduct Committee 🛡️

We have a code of conduct in place to make sure that the conference is a safe
place for many different people. This team is responsible for receiving and
handling CoC reports:

- Vicky Twomey-Lee
- Cheuk Ting Ho
- Georgi Ker
- Jakub Vysoký
- Michał Karzyński

## PyLadies Poland 👩‍💻

- Dorota Ostrowska
- Agata Skamruk
- Anna Wszeborowska

## Pyladies Lunch 🥗

Bug hugs to Jodie Burchell for hosting the PyLadies lunch, interview sessions
with sponsors, and for a second year in a row managing all the Beginners Days
activities.

> ## PYO 🐍

Reviewer: I'd prefer that tbh -> people can understand what's for
```suggestion
## Community Organizers Summit 🐍
```

Reviewer 2: also include the other summits?
<https://ep2026.europython.eu/language-summit/>
<https://ep2026.europython.eu/session/rust-summit-at-europython>
<https://ep2026.europython.eu/session/packaging-summit>

support this year too as our event manager.

She handled communication with all suppliers, relieving volunteers and board
members of logistical tasks. She coordinated with the venue, catering, speakers'
dinner, social events, sprints venue, booth builders, t-shirts, stickers, and
numerous other smaller suppliers.

We are deeply grateful to Anežka for her invaluable work and all the support she
provided throughout the year.

## Programme 📅

This team manages the Call for Proposals, Community Voting, Proposals Screening,
Panels, final selection of the Talks, Tutorials, Posters, Scheduling, Keynotes,
Open Spaces, and all the last minute cancellations 😅.

- Marina Moro Lopez
- Maria José Molina Contreras
- Naa Nortey
- Diego Russo
- Rodrigo Girão Serrão
- Cristián Maureira-Fredes

## Speaker Mentorship 🎓

In parallel with the Call for Proposals, we kick off the Speaker Mentorship
Programme. It's designed to help new and less experienced speakers submit
proposals and, if accepted, build their confidence and prepare their talks.

- Naa Nortey

## Communications & Design 📧

The Communications & Design team works year-round, with significant efforts

> during the conference launch, and major deadlines, like CFP, tickets, etc.

Reviewer 3: ```suggestion
The Communications & Design team works year-round, however they are at their busiest
during the conference week itself. Daily updates, monthly newsletters, WiFi passwords,
venue maps, social media posts, and even website copy are all things produced by this
incredible team of heroes.
```

This team is responsible for EuroPython’s social media accounts, blog,
newsletter, media partnerships, overall communications, as well as digital and

> printed designs throughout the year:

Reviewer 3: ```suggestion
This team is also responsible for the EuroPython social media accounts: the design team will
produce high quality artwork to go along with the copy written by the Communications boffins.

There are quite a few more forms of communication that we didn't yet mention. Unsurprisingly,
they all come under the Communications remit, too!
```

- Daria Linhart Grudzien
- Mia Bajić
- Celina Czyszczon
- Andrew Northall
- Sangarshanan Veera
- Daksh P. Jain

> ### Website, Automation & More 💻

Reviewer 3: gotta give it  a better title tbh

Author: Hmm do you have any suggestions?

Reviewer 2: There's an Infrastructure team at https://www.europython-society.org/europython-teams/ ?

```suggestion
### Infrastructure 💻
```

---
title: Thank You

> subtitle: For the EuroPython 2026 Team

Reviewer 2: ```suggestion
subtitle: To the EuroPython 2026 Team
```

---

# Thank You 🤗

EuroPython has been organised by volunteers since 2002, driven by the passion
and dedication of the community.

What does it take to organise a conference? Let's go through it chronologically
and tell you a bit more about the efforts and people behind it.

## EuroPython Society Board 🏛️

Before the conference preparations even start, a board is elected annually. The
board manages the day-to-day operations of the EPS, the non-profit behind the
series. They're responsible for fiscal and legal work, as well as making the
conference happen. That includes picking a location, finding a suitable venue,
local lawyers and accountants, and finally building teams.

> The board finds leads and builds teams of volunteers to work in different areas.

Reviewer 2: Remove duplication:

```suggestion
local lawyers and accountants.

Finally, the board finds leads and builds teams of volunteers to work in different areas.
```

The board members for the 2026 conference are:

- Anders Hammarquist
- Angel Ramboi
- Aris Nivorlis
- Artur Czepiel
- Mia Bajić
- Ege Akman
- Yuliia Barabash

## Team Anežka 💪

- Anežka Müller

Following our successful collaboration last year, Anežka continued to provide
support this year too as our event manager.

She handled communication with all suppliers, relieving volunteers and board
members of logistical tasks. She coordinated with the venue, catering, speakers'
dinner, social events, sprints venue, booth builders, t-shirts, stickers, and
numerous other smaller suppliers.

We are deeply grateful to Anežka for her invaluable work and all the support she
provided throughout the year.

## Programme 📅

This team manages the Call for Proposals, Community Voting, Proposals Screening,
Panels, final selection of the Talks, Tutorials, Posters, Scheduling, Keynotes,
Open Spaces, and all the last minute cancellations 😅.

- Marina Moro Lopez
- Maria José Molina Contreras
- Naa Nortey
- Diego Russo
- Rodrigo Girão Serrão

> - Cristián Maureira-Fredes

Reviewer 4: Cristián was the lead.

## Speaker Mentorship 🎓

In parallel with the Call for Proposals, we kick off the Speaker Mentorship
Programme. It's designed to help new and less experienced speakers submit
proposals and, if accepted, build their confidence and prepare their talks.

- Naa Nortey

## Communications & Design 📧

The Communications & Design team works year-round, however they are at their
busiest during the conference week itself. Daily updates, monthly newsletters,

> WiFi passwords, venue maps, social media posts, and even website copy are all

Reviewer 2: ```suggestion
Wi-Fi passwords, venue maps, social media posts, and even website copy are all
```

communications, and ensures invoices are submitted:

- Doreen Peace Nangira Wanyama
- Theofanis Petkos
- Raquel Dou
- Aris Nivorlis
- Panagiotis Skias

## Operations ⚙️

This is the largest team, divided into several smaller teams. You might have met
them onsite or online when contacting our helpdesk.

Operations Team:

- Jake Balas
- George Zisopoulos
- Janusz Kamieński
- Liberta Gani
- Mahe Iram Khan
- Moisés Guimarães
- Martin Borus
- Niklas Mertsch
- Panagiotis Kyrillos
- Piotr Gnus
- Raquel Dou
- Çınar Fidanboy
- Aeneas Christodoulou
- Toomas Ormisson
- Szymon Cader
- Jakub Červinka

## Code of Conduct Committee 🛡️

We have a code of conduct in place to make sure that the conference is a safe
place for many different people. This team is responsible for receiving and
handling CoC reports:

- Vicky Twomey-Lee
- Cheuk Ting Ho
- Georgi Ker
- Jakub Vysoký
- Michał Karzyński

## PyLadies Poland 👩‍💻

- Dorota Ostrowska
- Agata Skamruk
- Anna Wszeborowska

## PyLadies Lunch 🥗

Big hugs to Jodie Burchell for hosting the PyLadies lunch, interview sessions
with sponsors, and for a second year in a row managing all the Beginners Days

> activities.

Reviewer 5: ```suggestion
Big hugs to the organizers for hosting the PyLadies lunch, Open Space, Workshops, and more!.

- Anwesha Das
- Jodie Burchell
- Maria José Molina Contreras

```

> ## Community Organisers Summit 🐍

Reviewer 5: ```suggestion
## Beginners' Day

Thank you for the organization of the Humble Data and Django Girls Workshop at the sprints days of the conference:

- Jodie Burchell
- Raffaella Suardini 

## Community Organisers Summit 🐍
```

Reviewer 2: And "DjangoGirls" -> "Django Girls"

https://djangogirls.org/en/
